README.md

```diff
@@ -183,6 +183,14 @@ Meeting notes are held in a [Google doc](https://docs.google.com/document/d/18JV
 
 You can also join us on [gitter](https://gitter.im/open-telemetry/opentelemetry-erlang).
 
+### Java Auto Instrumentation
```

Comment:
For future browsers of this document, I think it would be helpful to make this more discoverable to those interested in Java... as such, how about moving this up to line 152 (i.e., just before the Javascript SDK) and doing four `####`s to create additional nesting?